.env-sample

```diff
@@ -1,4 +1,8 @@
 //N.B: When writting the below credentials ommit the curl braces.
 
 DB_URL=postgres://{db_username}:{db_password}@{host}:{port}/{db_name}
-TOKEN_SECRET= "your_secret"
\ No newline at end of file
+TOKEN_SECRET= "your_secret"
+POSTGRES_USER={db_user}
+POSTGRES_PASSWORD={db_password}
+POSTGRES_DB={db_name}
+DB_URL_CONTAINER=postgres://{db_username}:{db_password}@db:{port}/{db_name}
```

Comment:
How is this different from `DB_URL`?

Reply:
They are different because its where you told me to replace the host with the service name in docker-compose

Reply:
You can use the same `DB_URL`. What needs to change is the host. You should instructions in the readme file to this effect